<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalorieCalcWindow</class>
 <widget class="QDialog" name="CalorieCalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_dishCalc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_addProduct">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008080;
color: white;</string>
     </property>
     <property name="text">
      <string>Додати продукт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_saveDish">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>390</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008080;
color: white;</string>
     </property>
     <property name="text">
      <string>Зберегти Страву</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dishName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_dishName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть назву страви</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nameProductToAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_guantityOfProduct">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_prodName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Назва</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prodCount">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Кількість(г)</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_dishProduct">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dishDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_dishDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Додайте опис страви</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_customDish">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
